.github/workflows/pr-checks.yml

```diff
@@ -0,0 +1,46 @@
+name: "PR checks"
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Do you think this needs to run when pushing to main?

Reply:
Thanks, removed